java/feature-layer-dictionary-renderer/README.md

```diff
@@ -0,0 +1,47 @@
+# Dictionary Renderer with Feature Layer
+
+Demonstrates how to display a FeatureLayer with military symbology, using ArcGIS Runtime's DictionaryRenderer.
+
+![Dictionary Renderer with Feature Layer App](feature-layer-dictionary-renderer.png)
+
+## How it works
+
+This sample loads a number of point, line, and polygon feature tables from a Runtime geodatabase. For each feature table, a FeatureLayer is created, and a DictionaryRenderer object is created and applied to the layer. Note that each layer needs its own renderer, though all renderers can share the DictionarySymbolStyle, in which case all layers will use the same symbology specification (MIL-STD-2525D in the case of this sample). Each layer is added to the map, and when all layers are loaded, the map's viewpoint is set to zoom to the full extent of all feature layers.
+
+
+## Relevant API
+
+* ArcGISMap
+* Basemap
```

Comment:
Lets pair this down a bit and restrict it to the classes specifically being demonstrated in this sample.

- DictionaryRenderer
- DictionarySymbolStyle
- FeatureLayer

Reply:
Sure, I added geodatabase too, to this list.